--- Makefile	2018-03-18 18:09:33.000000000 +0530
+++ Makefile2	2018-03-18 20:18:42.000000000 +0530
@@ -25,8 +25,6 @@
 DESTDIR?=/usr
 PREFIX?=/local
 
-LDCONFIG?=ldconfig
-
 ifneq ($V,1)
 Q ?= @
 endif
@@ -107,12 +105,12 @@
 .PHONY:	install-deb
 install-deb:	$(DYNAMIC)
 	$Q echo "[Install Headers: deb]"
-	$Q install -m 0755 -d							~/wiringPi/debian-template/wiringPi/usr/include
-	$Q install -m 0644 $(HEADERS)						~/wiringPi/debian-template/wiringPi/usr/include
+	$Q install -m 0755 -d							~/wiringPi/debian/wiringPi/usr/include
+	$Q install -m 0644 $(HEADERS)						~/wiringPi/debian/wiringPi/usr/include
 	$Q echo "[Install Dynamic Lib: deb]"
-	install -m 0755 -d							~/wiringPi/debian-template/wiringPi/usr/lib
-	install -m 0755 libwiringPi.so.$(VERSION)				~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPi.so.$(VERSION)
-	ln -sf ~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPi.so.$(VERSION)	~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPi.so
+	install -m 0755 -d							~/wiringPi/debian/wiringPi/usr/lib
+	install -m 0755 libwiringPi.so.$(VERSION)				~/wiringPi/debian/wiringPi/usr/lib/libwiringPi.so.$(VERSION)
+	ln -sf ~/wiringPi/debian/wiringPi/usr/lib/libwiringPi.so.$(VERSION)	~/wiringPi/debian/wiringPi/usr/lib/libwiringPi.so
 
 .PHONY:	uninstall
 uninstall:
